packages/rope/rope.0.6/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+authors: [ "Christophe Troestler" ]
+license: "LGPL-2.1 with OCaml linking exception"
+homepage: "https://github.com/Chris00/ocaml-rope"
+dev-repo: "https://github.com/Chris00/ocaml-rope.git"
+bug-reports: "https://github.com/Chris00/ocaml-rope/issues"
+doc: "https://Chris00.github.io/ocaml-rope/doc"
+tags: [ "datastructure"  ]
+build: [
+  [ "jbuilder" "subst" ] {pinned}
+  [ "jbuilder" "build" "-p" name "-j" jobs ]
+]
+build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
+depends: [
+  "base-bytes"
+  "jbuilder" {build}
+  "benchmark" {test}
+  "ocamlfind" {build & >= "1.5"}
```

Comment:
I wonder why ocamlfind is a requirement here. If you're using jbuilder, you're not really using ocamlfind directly.